README.md

```diff
@@ -41,28 +41,26 @@ $ sudo gem install bourbon
 $ sudo gem install neat
 
 # Then, navigate to the directory to install Bourbon into
-$ cd /scaffold/libraries
+$ cd path/to/scaffold/libraries
 $ bourbon install
 $ neat install
 ```
 
 ### Compile manually
-While working on the Scaffold, compile your CSS and test the output by running it in this folder. Here's the command: 
+While working on the Scaffold, compile your CSS and test the output by running it in this folder. Here's the command:
 
 ```sh
-$ sass --watch styles.scss:output.css --style compressed
+$ sass --watch path/to/styles.scss:path/to/output.css --style compressed
 ```
 
-### Run Lint
-We use the Brigade Linter gem: <https://github.com/brigade/scss-lint>
+### Linting
+Linting on the scaffold uses [pre-commit by Yelp](http://pre-commit.com/) software.
```

Comment:
For our SCSS testing we are using the linter `[pre-commit by Yelp](http://pre-commit.com/)` software.

I wasn't to use the word testing in this sentence, for when we have new employees and they might not know the term "linting"